jobrunner/config.py

```diff
@@ -60,7 +62,8 @@ class ConfigException(Exception):
     "python",
 }
 
-DOCKER_REGISTRY = "ghcr.io/opensafely-core"
+# DOCKER_REGISTRY = "ghcr.io/opensafely-core"
+DOCKER_REGISTRY = os.environ.get("DOCKER_REGISTRY", "")
```

Comment:
@bloodearnest We'll need to provide this in other environments, won't we?

Reply:
Yeah, we need this, but the default should be "ghcr.io/opensafely-core" if the env is not defined I think

Reply:
Sure, will make ghcr.io/opensafely-core as default